src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1041,11 +1037,23 @@ public LocalDate fromString(String string) {
           checkBoxCvrTreatBlankAsUndeclaredWriteIn
               .setSelected(ContestConfig.SUGGESTED_TREAT_BLANK_AS_UNDECLARED_WRITE_IN);
         }
+        case CSV -> {
+          buttonAddCvrFile.setDisable(false);
+          textFieldCvrFilePath.setDisable(false);
+          buttonCvrFilePath.setDisable(false);
+          textFieldCvrContestId.setDisable(true);
```

Comment:
It looks like you use `contentId` in `CsvCvrReader`, so you probably want this set to `false` instead.

Reply:
It was set but had no impact on the output files, so @tarheel noted it should be removed. I removed all references to it from `CsvCvrReader` too.